src/app/public/modules/autonumeric/autonumeric-options-provider.ts

```diff
@@ -7,13 +7,14 @@ import {
 } from './autonumeric-options';
 
 /**
- * Provides options to the underlying [`autoNumeric` utility](https://github.com/autoNumeric/autoNumeric).
+ * Provides options to the underlying [autoNumeric utility](https://github.com/autoNumeric/autoNumeric).
+ * This may be used to set global options on multiple input fields.
  */
 @Injectable()
 export class SkyAutonumericOptionsProvider {
 
   /**
-   * Specifies the value for a settings object to pass to the [`autoNumeric` utility](https://github.com/autoNumeric/autoNumeric).
+   * Specifies the value for a settings object to pass to the [autoNumeric utility](https://github.com/autoNumeric/autoNumeric).
```

Comment:
```suggestion
   * Specifies the value for a settings object to pass to the autoNumeric utility.
```
Like you pointed out, we probably have more links than we need. Given that we have one in the service description before this method description, I'd dump this link.